docker/oso-saml-sso/centos7/root/config.yml

```diff
@@ -105,3 +105,11 @@
         content: "{{ saml2_sso_configdata.saml2_idp_cert_data.cert }}"
       - name: "{{ saml2_sso_configdata.saml_privatekey_file }}"
         content: "{{ saml2_sso_configdata.saml2_idp_cert_data.key }}"
+
+  # NOTE: this must be the final task because it indicates to
+  #       the start-up script that this playbook successfully
+  #       ran to completion.
```

Comment:
Is it worth an additional comment to say "ansible exits with successful return code even if it times out"?

The code is clear in what it does, but it takes institutional knowledge to know WHY it's here.